src/odemis/gui/model/tab_gui_data.py

```diff
@@ -66,6 +69,10 @@
 from odemis.util.filename import create_filename, make_unique_name
 
 
+class AcquiMode(Enum): # TODO: move to better place
```

Comment:
I feel it's a very generic name for a very specific purpose... but I don't have really better proposal. Could you at least use a full words `AcquisitionMode`. 
Also, the right place for it should be in gui/model/_constant.py

An alternative could be to just use `model.MD_AT_FLUO` and `model.MD_AT_FIB`.